raft/handler.go

```diff
@@ -9,18 +9,19 @@ import (
 	"strconv"
 )
 
-// HTTPHandler represents an HTTP endpoint for Raft to communicate over.
-type HTTPHandler struct {
-	log *Log
-}
-
-// NewHTTPHandler returns a new instance of HTTPHandler associated with a log.
-func NewHTTPHandler(log *Log) *HTTPHandler {
-	return &HTTPHandler{log: log}
+// Handler represents an HTTP endpoint for Raft to communicate over.
+type Handler struct {
+	Log interface {
```

Comment:
It's because the interface is specific to the `Handler` and not used elsewhere. I could make a separate interface called `HandlerLog` but that seems pointless (and kind of ugly naming).